src/zkevm_specs/util/param.py

```diff
@@ -76,3 +76,9 @@
 SSTORE_SET_GAS = 20000
 SSTORE_RESET_GAS = 2900
 SSTORE_CLEARS_SCHEDULE = 15000
+
+# log static gas
+LOG_STATIC_GAS = 375
```

Comment:
```suggestion
GAS_COST_LOG = 375
```

and move to above section, like line 46